### ISSUE TITLE
Better logging on failed backups

### ISSUE THREAD

**Issue by FuzzyMistborn** [cupcakearmy/autorestic#49]
**Describe the bug**

Running `autorestic backup -a` or even just the name of the location only seems to run the first of the "to" backends listed.  The second does not run.

**Expected behavior**

Multiple locations are backed up.

**Environment**
 - OS: [e.g. iOS] Ubuntu
 - Version: [e.g. 22] 1.0.3

**Additional context**

Here's my config:
```
backends:
  synology_docker:
    type: sftp
    path: {{main_username}}@synology:/Restic/
    key: {{ secret_restic_repo_password }}
    env: {}
  b2_docker:
    type: b2
    path: '{{ secret_restic_b2_bucket }}:/'
    key: {{ secret_restic_repo_password }}
    env:
      B2_ACCOUNT_ID: {{ secret_restic_b2_account_id }}
      B2_ACCOUNT_KEY: {{ secret_restic_b2_account_key }}
locations:
  docker:
    from: '~/docker'
    to: 
      - synology_docker
      - b2_docker
    options:
      forget:
        keep-last: 30
```

**Comment by cupcakearmy** [comment]
That is definitely not the way it should work. Can you check the complete output?

```bash
autorestic backup -av
```

**Comment by FuzzyMistborn** [comment]
Yeah, I thought of that after I posted.  It's a permissions error where some of the folders are owned by different users (ie my docker dir has some stuff owned by say....root or www-data and is read-only to the user).  Any chance the logging could be improved on just -a to make clear it's failing?  Because it _looked_ like it was succeeding but it was not.

**Comment by FuzzyMistborn** [comment]
Is there a way to run autorestic with a custom restic binary as described here? https://restic.readthedocs.io/en/stable/080_examples.html#full-backup-without-root

**Comment by cupcakearmy** [comment]
> Is there a way to run autorestic with a custom restic binary as described here? https://restic.readthedocs.io/en/stable/080_examples.html#full-backup-without-root ? Just run autorestic as the `restic` user? (Sorry for the dumb question, I'm tired after a long night trying to fix some stuff and wanting my backups running again).

😅 I can relate to that. Devops is always super tiring 😴.
Absolutely, it should definetely tell you that something went wrong, as otherwise the user gets a false sense of security, which is super bad.

About the custom binary:
In theory one could add a flag that points to that binary, and/or a config option so you don't have to always specify it.
Need to check the go docs how path is resolved and execute a specific binary, but I'm sure it's possible

**Comment by cupcakearmy** [comment]
I've opened another ticket for the custom binary @FuzzyMistborn

**Comment by FuzzyMistborn** [comment]
Yeah I think if there's an error outputting the error to the terminal would be useful.  Like for me I was getting this running verbose:

```
Backend: synology_docker_beta
> Executing: /usr/bin/restic backup .
error: open .storage/auth: permission denied
error: open .storage/core.config: permission denied
```

And thanks on the custom binary.  I'll dig into this a bit more.

**Comment by cupcakearmy** [comment]
Yes, if the command errored it's imperative that the output is shown, also the script should exit 1 at the end, so if it runs in a CI the system sees it as failed

**Comment by cupcakearmy** [comment]
Will be released in 1.0.5
